### ISSUE TITLE
Not working without setting 'at' and problem with multiple notifications after updating to 3.0.0

### ISSUE THREAD

**Issue by sowrensen** [EddyVerbruggen/nativescript-local-notifications#113]
As the title says, after the update notifications are not showing without setting the `at`. And if I set a time however, only the last one comes up from the array. Everything was working just before the update.

**Comment by Danziger** [comment]
@sowrensen The first issue you mention has been fixed in #109. Could you please try to upgrade to `3.0.1`?

Regarding the second one, I will soon open a PR for another issue that might also fix this: https://github.com/Danziger/nativescript-local-notifications/pull/5

Meanwhile, could you please try to set a different `id` to each notification you schedule? Once my PR is merged, the plugin will take care of that automatically.

**Comment by sowrensen** [comment]
@Danziger Thanks. I've updated to 3.0.1 and everything is working perfectly. Just FYI, I disabled the ID on 3.0.0 cause the plugin was not working when I set `Date.now()` as the ID. Now I changed that to integer and seems the application is alright.
